.eslintrc.js

```diff
@@ -10,11 +15,32 @@ module.exports = {
 	globals: {
 		asyncRequire: true
 	},
+	plugins: [
+		'import'
+	],
+	settings: {
+		'import/resolver': {
+			[ customResolverPath ]: {}
+		}
+	},
 	rules: {
 		camelcase: 0, // REST API objects include underscores
+		// NOTE: Some import rules are errors in client (Webpack's resolution) and
+		//       warnings in server (Node's module resolution).
```

Comment:
Yes, we can define all of these rules as errors and override them inside `server/.eslintrc.json`. The following rules are already overridden for the server: 
- Set to warn:
  - `import/default`
  - `import/export`
  - `import/named`
  - `import/namespace`
- Set to ignore:
  -` import/no-commonjs`

Which rules would you recommend upgrading to error in `.eslintrc.js`? The following rules are currently set as warnings for the client: 
- `import/no-commonjs`
- `import/no-named-as-default-member`
- `import/no-named-as-default`
- `import/no-webpack-loader-syntax`